doc/mlaunch.rst

```diff
@@ -99,6 +99,20 @@ Usage
                 [--hostname HOSTNAME] [--auth] [--username USERNAME]
                 [--password PASSWORD] [--auth-db DB]
                 [--auth-roles [ROLE [ROLE ...]]]
+                [--initial-user] [--sslCAFile SSLCAFILE]
+                [--sslCRLFile SSLCRLFILE] [--sslAllowInvalidHostnames]
+                [--sslAllowInvalidCertificates] [--sslOnNormalPorts]
+                [--sslMode {disabled,allowSSL,preferSSL,requireSSL}]
+                [--sslPEMKeyFile SSLPEMKEYFILE]
+                [--sslPEMKeyPassword SSLPEMKEYPASSWORD]
+                [--sslClusterFile SSLCLUSTERFILE]
+                [--sslClusterPassword SSLCLUSTERPASSWORD]
+                [--sslDisabledProtocols SSLDISABLEDPROTOCOLS]
+                [--sslWeakCertificateValidation]
```

Comment:
This option is replaced with `sslAllowConnectionsWithoutCertificates` since v3.0.